src/common/constants.ts

```diff
@@ -61,6 +61,18 @@ export const CONNECTION_RECONNECT_ATTEMPTS = 5;
 
 export const MEDIA_MESSAGE_TYPES = ['IMAGE', 'AUDIO', 'VIDEO', 'DOCUMENT', 'STICKER'];
 
+export const getObject = (arr: any, data: any) => {
```

Comment:
This does not belong to constants. Can you create common/utils.ts and move this function there.